src/api/profileChips.js

```diff
@@ -1,17 +1,18 @@
-import fetch from 'node-fetch';
+import wrappedFetch from './wrappedFetch';
 
 const getProfileChips = async () => {
-  const url = '/api/v1/identities';
+  // const url = new URL(process.env.REACT_APP_API_HOST);
+  // TODO: Not getting the data back with the URL env variable, need to update this url
+  const url = 'http://127.0.0.1:8080/api/v1/identities';
```

Comment:
Is this correct? I see the data 
![image](https://user-images.githubusercontent.com/1613526/114793486-7923bc00-9d58-11eb-95c1-bfa8452dd10b.png)